editor/html-template.ts

```diff
@@ -8,17 +8,27 @@ export const getHtmlTemplate = (spec: string) => `
 <html>
 <head>
     <title>Gosling Visualization</title>
-    <link rel="stylesheet" href="https://unpkg.com/higlass@${higlassVersion}/dist/hglib.css">
-    <script src="https://unpkg.com/react@${reactVersion}/umd/react.production.min.js"></script>
-    <script src="https://unpkg.com/react-dom@${reactVersion}/umd/react-dom.production.min.js"></script>
-    <script src="https://unpkg.com/pixi.js@${pixiVersion}/dist/browser/pixi.min.js"></script>
-    <script src="https://unpkg.com/higlass@${higlassVersion}/dist/hglib.js"></script>
-    <script src="https://unpkg.com/gosling.js@${goslingVersion}/dist/gosling.js"></script>
+    <link rel="stylesheet" href="https://esm.sh/higlass@${higlassVersion}/dist/hglib.css">
+
+    <script type="importmap">
+    {
+    "imports": {
+        "react": "https://esm.sh/react@${reactVersion}?bundle",
+        "react-dom": "https://esm.sh/react-dom@${reactVersion}?bundle",
+        "pixi.js": "https://esm.sh/pixi.js@${pixiVersion}?bundle",
+        "higlass": "https://esm.sh/higlass@${higlassVersion}?bundle",
+        "gosling.js": "https://esm.sh/gosling.js@${goslingVersion}?bundle"
```

Comment:
With `?bundle` I'm actually not sure gosling.js has any imports to resolve. I would be curious about just using:

```javascriprt
        import { embed } from 'https://esm.sh/gosling.js?bundle';
```

below. The main issue I issue I see with this is that I'm not sure how to control the version of HiGlass so that it aligns with the css.

Reply:
esm.sh says 
> In bundle mode, all dependencies are bundled into a single JS file except the peer dependencies.

So actually I don't need to import Higlass! Just react, react-dom, and pixi.js